### ISSUE TITLE
Add "draft" status.

### ISSUE THREAD

**Issue by joshuapinter** [r0adkll/upload-google-play#59]
Adding support for a "draft" status would be immensely useful. I believe a lot of people want to make some final changes or tests before actually releasing the new version, including updating the Release Notes.

I'm not very familiar with GitHub Actions but after a quick look, I believe just adding support for a `status` option here should do the trick: 

https://github.com/r0adkll/upload-google-play/blob/master/lib/edits.js#L179

If I can help test or even if you need me to do a PR, let me know.

Thanks!

**Comment by cWenyu** [comment]
Hi @joshuapinter, does the package allow to use draft status? If not, could you tell me how did you solve the problem? 

Thanks 😺 .

**Comment by joshuapinter** [comment]
@cWenyu It doesn't. I still want/need this. You can create a PR for this if you need it as well.

**Comment by cWenyu** [comment]
@joshuapinter  Thank you for your reply, I solved the issue by promoting internal testing to the next phase and the staus went from "draft" to "In review". I know it might be wrong, but I think deploying the app via firebase is quite time-consuming, change status is a quicker way and it meets my current requirements.

**Comment by joshuapinter** [comment]
We’re doing the same thin as a workaround. Being able to set a draft status is definitely the way to go though.

**Comment by auriorx** [comment]
@joshuapinter I think I've found a work-around.

`
if (options.userFraction != undefined) {        status = 'inProgress';    } else {        status = 'completed';    }
`

I saw this piece of code in the source. The status of the release will not be set to completed if userFraction is defined. So for me, using userFraction: 0.99 worked!

I hope this little workaround will suit your needs.

**Comment by daniil-shumko** [comment]
None of the workarounds were good enough for me so I've decided to use `fastlane supply` instead of this step. Fastlane comes pre installed on GitHub Actions anyway and `fastlane supply` allows you to specify what status you want to use when deploying the app.

I was planing to move to Fastlane in the future anyway but seen that this step is not actively maintained made be migrate earlier than i wanted.

**Comment by auriorx** [comment]
@daniil-shumko thanks for mentioning that repo! I will definitely be checking that out too.

Might also be interesting for @PedroBarbosaPT.

**Comment by daniil-shumko** [comment]
@auriorx no worries. Here is a quick command that I am using that replaces this whole step for me.
```
fastlane supply --apk 'path/to/apk' --json_key_data 'json key string' --package_name 'com.example.app' --track 'internal' --release_status 'draft'
```
If you need to pass any other parameters or if you are using a JSON file instead of JSON string from the Secrets, you can find all the [documentation on fastlanes website](https://docs.fastlane.tools/actions/supply/#parameters).

**Comment by auriorx** [comment]
@daniil-shumko I'm very grateful, thanks again!

**Comment by ModdyLP** [comment]
If I use the fixes from the comments i get another error, may someone can help.
'Error: Staged rollout is not permitted on draft app.'

**Comment by r0adkll** [comment]
Should now be resolved in `v1.0.16` per PR #78
